platforms/hermes-kotlin/hermes-kotlin-test/src/test/kotlin/FfiTest.kt

```diff
@@ -1,21 +1,5 @@
-import ai.snips.hermes.AsrDecodingDuration
-import ai.snips.hermes.AsrToken
-import ai.snips.hermes.AsrTokenRange
-import ai.snips.hermes.ContinueSessionMessage
-import ai.snips.hermes.DialogueConfigureIntent
-import ai.snips.hermes.DialogueConfigureMessage
-import ai.snips.hermes.EndSessionMessage
+import ai.snips.hermes.*
```

Comment:
do not use .* imports, there is a setting in intellij to change the threshold for this